src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -679,7 +680,7 @@ uint64_t Visitor::visit(Parser::TextLimitClauseContext* ctx) {
 SparqlValues Visitor::visit(Parser::InlineDataOneVarContext* ctx) {
   SparqlValues values;
   auto var = visit(ctx->var());
-  values._variables.push_back(var.name());
+  values._variables.push_back(Variable{var.name()});
```

Comment:
Doesn't the `visit` directly return a `Variable` and you can write
`values._variables.push_back(visit(ctx->var());`
(The toString and then toVariable again is unneeded.

Reply:
Indeed :-)